uxarray/plot/accessor.py

```diff
@@ -77,11 +87,32 @@ def points(self, element="nodes", backend=None, **kwargs):
         else:
             raise ValueError(f"Unsupported element {element}")
 
+        # Set default kwargs for better visualization
+        if "color" not in kwargs:
+            kwargs["color"] = "blue"
+        if "size" not in kwargs:
+            kwargs["size"] = 50
+        if "title" not in kwargs:
+            if self._uxgrid.n_face == 0 and self._uxgrid.n_edge == 0:
+                kwargs["title"] = "Points"
+            else:
+                kwargs["title"] = f"{element.capitalize()}"
```

Comment:
A few things: 
- With these new defaults here, are we bypassing the defaults of `points_df.hvplot.points()`? 
- I don't know if we need to bypass them, but if you think that it contributes to a better plot, I'd at least suggest that we read these from constants.py or something similar to that maybe for plotting constants rather than hard-coding into the function. 
- Not sure providing a default title would help or confuse though

Reply:
Right observation, I got carried away in the notebook that I was curating for this PR. The idea was to have some defaults that would improve the out-of-box experience. Another approach: we strip back to just the essential geographic defaults and let hvplot handle the rest? - this would be cleaner.

Reply:
As long as the code is clean and readable, and duplicate code is avoided, either would be fine.